### PR TITLE
Set testsource to blank on selecting generator for openMrs Id

### PR DESCRIPTION
**Pull request by walteronoh**
### Description
Set "Test Source" to blank on selecting "Generator for openMrs Id".
[MF-656](https://issues.openmrs.org/browse/MF-656).

mf-656-fix_bug_switching_testsource_generator_for_openMrs

### CODE REVIEW

File: packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx [hunk 1]
```diff
@@ -62,6 +63,7 @@ export const IdentifierInput: React.FC<IdentifierInputProps> = ({ identifierType
 
         if (selectedSource.autoGenerationOption.automaticGenerationEnabled) {
           identifierType.autoGenerationSource = selectedSource;
+          setFieldValue(name, '');
```
Comment (HerbertYiga) [inline packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx:66]: thanks cc @FlorianRappl

File: packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ interface IdentifierInputProps {
 }
 
 export const IdentifierInput: React.FC<IdentifierInputProps> = ({ identifierType }) => {
+  const { setFieldValue } = useContext(PatientRegistrationContext);
```
Comment (manuelroemer) [inline packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx:17]: Can you merge this line with line 28 (`const { validationSchema, setValidationSchema } = React.useContext(PatientRegistrationContext);`)? There's no reason to use the same context twice. Ideally line 28 is removed (you can just add `validationSchema` and `setValidationSchema` to line 17.
Thanks!

**Review by manuelroemer — COMMENTED** [review]
Hey, thanks a lot, looks great! 🍻
Only one minor thing about a duplicate `useContext`, but then we can merge it!

**Review by manuelroemer — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**